commands/common_args.py

```diff
@@ -24,11 +24,11 @@ class Options:
         help="Kili API Key",
     )
 
-    model_framework = click.option(
-        "--model-framework",
+    tensor_backend = click.option(
```

Comment:
`ml_backend` (it contains tensors and auto differentiation, so more than tensors)